docs/HACKING.md

```diff
@@ -1,22 +1,50 @@
 # Hacking
 
-- Project conventions
-  - [TypeScript](#Typescript)
-    - [Dealing with problems](#dealing-with-problems)
-- Development setup
-  - [Development setup](#development-setup)
-  - [Running a branch](#running-a-branch)
-  - [Setting up or connecting to a MAAS](#maas-deployments)
-  - [Creating a Multipass instance](#creating-a-multipass-instance)
-  - [Creating a LXD instance](#creating-a-lxd-instance)
-  - [Building a production bundle](#building)
-  - [Creating a fake windows image](#creating-a-fake-windows-image)
-  - [Show intro](#show-intro)
-  - [Sample data](#sample-data)
-- Testing
-  - [Integration tests](#integration-tests)
-  - [Unit tests](#unit-tests)
-  - [Performance tests](#performance-tests)
+-   [Project conventions](#project-conventions){#toc-project-conventions}
```

Comment:
Why do all of these headings have {#toc-something} afterwards? I don't think this is normal syntax, and it shows up in the file preview:

![image](https://github.com/user-attachments/assets/068a66a0-8ec0-4e6e-9153-fd72ee1c9df6)